### DISCUSSION TITLE
Notion, Excalidraw or Insomnia would be nice :D

### DISCUSSION THREAD

**Discussion by lethamburn** [dr-slash/icons-flat-osx#104] [Requests]
Notion, Excalidraw or Insomnia would be nice :D

**Comment by lucky-yoolk** [comment]
> Notion, Excalidraw or Insomnia would be nice :D

Done ;)

**Reply by lethamburn** [reply]
Thank you very very very much!

**Reply by lethamburn** [reply]
Wich one is the best way to support all this thing? your work is amazing!
